pkg/policy/api/icmp.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import "strconv"
+
+type ICMPRules []ICMPRule
+
+// ICMPRule is a list of ICMP-type.
+type ICMPRule struct {
+	// Types is a list of ICMP-type.
+	//
+	// +kubebuilder:validation:Optional
+	Types []ICMPType `json:"types,omitempty"`
+}
+
+// ICMPType is a ICMP-type.
+type ICMPType struct {
+	// Type is a ICMP-type.
+	// It should be 0-255 (8bit).
+	//
+	// +kubebuilder:validation:Maximum:=255
+	// +kubebuilder:validation:Minimum:=0
```

Comment:
Nit: I think the extra colon near the end is not needed. I'm actually a bit surprised that the code gen tool didn't panic or something :)
```suggestion
	// +kubebuilder:validation:Maximum=255
	// +kubebuilder:validation:Minimum=0
```

Reply:
It's correctly work without the colon as you say, but It seems the correct format is with the extra colon.
ref: https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
Ah interesting. Probably worth going with what is already consistent in the codebase

Reply:
Yes, I'll remove them :)